airflow/www/views.py

```diff
@@ -1029,7 +1029,6 @@ def blocked(self, session=None):
             for dag_id, active_dag_runs in dags:
                 max_active_runs = 0
                 dag = dagbag.get_dag(dag_id)
-                max_active_runs = dagbag.dags[dag_id].max_active_runs
```

Comment:
Why removing it here? I guess the next 2 lines make it works as expected.

Reply:
Yes, sometimes the dag is `None` (as suggested also by the if condition) and in that cases the `dag_id` key may not be present. In general I think  `get_dag` is the right way to get a DAG from dagbag.

Reply:
👍